pkg/volume/volume.go

```diff
@@ -103,3 +113,31 @@ func RenameDirectory(oldPath, newName string) (string, error) {
 	}
 	return newPath, nil
 }
+
+// ReadWriteModeFor returns the ReadWriteMode represented by the given boolean.
+// A value of true maps to ReadOnly, false to ReadWrite
+func ReadWriteModeFor(ro bool) ReadWriteMode {
```

Comment:
It is needed to bridge the ReadOnly boolean on volumes.  I don't think we want to expose `ReadWriteMode` to users.